packages/dds/tree/src/test/feature-libraries/sequence-field/sequenceChangeRebaser.spec.ts

```diff
@@ -107,26 +107,33 @@ describe("SequenceField - Rebaser Axioms", () => {
 
     describe("A ○ A⁻¹ === ε", () => {
         for (const [name, mark] of testMarks) {
-            if (name === "Delete") {
-                it.skip(`${name} ○ ${name}⁻¹ === ε`, () => {
-                    /**
-                     * These cases are currently disabled because the inverse of Delete
-                     * does not capture which node it is reviving.
-                     */
-                });
-            } else {
-                it(`${name} ○ ${name}⁻¹ === ε`, () => {
-                    const change = [mark];
-                    const taggedChange = tagChange(change, brand(1));
-                    const inv = SF.invert(taggedChange, TestChange.invert);
-                    const actual = SF.compose(
-                        [taggedChange, tagInverse(inv, taggedChange.revision)],
-                        TestChange.compose,
-                    );
-                    const delta = SF.sequenceFieldToDelta(actual, TestChange.toDelta, fakeRepair);
-                    assert.deepEqual(delta, []);
-                });
-            }
+            it(`${name} ○ ${name}⁻¹ === ε`, () => {
+                const change = [mark];
+                const taggedChange = tagChange(change, brand(1));
+                const inv = SF.invert(taggedChange, TestChange.invert);
+                const actual = SF.compose(
+                    [taggedChange, tagInverse(inv, taggedChange.revision)],
+                    TestChange.compose,
+                );
+                const delta = SF.sequenceFieldToDelta(actual, TestChange.toDelta, fakeRepair);
+                assert.deepEqual(delta, []);
+            });
+        }
+    });
+
+    describe("A⁻¹ ○ A === ε", () => {
```

Comment:
I was also thinking of adding such a suite, but now that I've thought about it more, it doesn't make sense to me. `A⁻¹ ○ A === ε` is not an axiom we expect/require to be true because `A⁻¹⁻¹ === A` is not always true (because `insert⁻¹⁻¹ === revive`). The fact that the current implementation passes this test is fine, but that's an implementation detail.